versions.props

```diff
@@ -40,7 +40,7 @@ org.jmock:jmock = 2.12.0
 org.knowm.xchart:xchart = 3.6.1
 com.palantir.conjure.verification:* = 0.19.0
 io.undertow:undertow-core = 2.2.18.Final
-com.squareup.okhttp3:mockwebserver = 3.13.1
+com.squareup.okhttp3:mockwebserver3-junit5 = 5.0.0-alpha.10
```

Comment:
There hasn't been a non-alpha release of `mockwebserver3-junit5` yet, but there have been many alpha releases over the past year and a half:

From https://search.maven.org/artifact/com.squareup.okhttp3/mockwebserver3-junit5

| Version | Release Date |
| ------- | ------------ |
| 5.0.0-alpha.10 | 27-Jun-2022 |
| 5.0.0-alpha.9 | 16-Jun-2022 |
| 5.0.0-alpha.8 | 08-Jun-2022 |
| 5.0.0-alpha.7 | 26-Apr-2022 |
| 5.0.0-alpha.6 | 14-Mar-2022 |
| 5.0.0-alpha.5 | 21-Feb-2022 |
| 5.0.0-alpha.4 | 01-Feb-2022 |
| 5.0.0-alpha.3 | 22-Nov-2021 |
| 5.0.0-alpha.2 | 30-Jan-2021 |